instrumentation/build.gradle.kts

```diff
@@ -1,58 +0,0 @@
-// this project will run in isolation under the agent's classloader
```

Comment:
I had accidentally renamed this without the intention of migrating and renamed back in this commit so looks like a deletion when looking at commits